src/Chunk.cpp

```diff
@@ -2278,6 +2299,38 @@ bool cChunk::DoWithBeaconAt(int a_BlockX, int a_BlockY, int a_BlockZ, cBeaconCal
 
 
 
+bool cChunk::DoWithBrewingstandAt(int a_BlockX, int a_BlockY, int a_BlockZ, cBrewingstandCallback & a_Callback)
+{
```

Comment:
I would like to use DoWithChunkAt, and GetBlockEntity. They provide enough info to get the blockentity.